cmasher/utils.py

```diff
@@ -233,6 +239,102 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
+def combine_cmaps(
+    cmaps: list[Union[Colormap, str]],
+    nodes: Optional[list[float]] = None,
+    n_rgb_levels: int = 256,
```

Comment:
Since there is no backward compatibility concerns for brand new API, I suggest to make it slightly more restrictive from the start by making `nodes` and `n_rgb_levels` keyword only.
```suggestion
    *,
    nodes: Optional[list[float]] = None,
    n_rgb_levels: int = 256,
```

Reply:
note that it would also allow to makes `cmaps` a collection of all positional arguments, which would (maybe ?) make the interface slightly lighter. For instance, this could be allowed
```python
combine_cmaps("Blues", "Greens")
```
what do you think ?

Reply:
I think it makes sense to require stricter args here, though it would not make any difference for current code (there is check following to avoid these two args being swapped), but it's always great to make it more robust.

For the second influence here, I didn't expect this to happen but it looks good.

Reply:
> For the second influence here, I didn't expect this to happen but it looks good.

it won't happen by itself, but that's a possibility if we first make all other arguements keyword only !
Specifically, we'd need to change the signature to something like
```python
def combine_cmaps(*cmaps, nodes=None, n_rgb_levels=256):
```
where I kept type annotations out for simplicity.

Reply:
I must have not expressed myself clear enough here. I meant to say "I didn't know such good thing would happen before", not "I don't think it would have such influence" :-).